metagraph-plugin/tests/test_algorithms.py

```diff
@@ -13,6 +13,13 @@ def test_bfs(networkx_weighted_directed_8_12, kg_from_nx_di_8_12):
     assert bfs2_kg.tolist() == [2, 4, 5, 6, 7]
 
 
+def test_bfs_iter(networkx_weighted_directed_bfs, kg_from_nx_di_bfs):
```

Comment:
if this is failing then you should mark it with xfail.

Reply:
Actually, this is the purpose of this test case. It would fail until the BFS implementation is fixed.

Reply:
Correct, but my suggestion is to include this test on our master.  The way you can include it is to mark it with xfail and then it will be running but not failing our CI.  Once this is fixed those that fix the issue can go ahead and enable this test (as otherwise they will not be able to have their CI green).

Reply:
Sure. I committed the changes to address both of your comments.

Reply:
Thanks.  Can you make sure to update your Jira ticket for this case and mention the unit test you included.

Reply:
They are already linked. I updated the issue with the xfail label.